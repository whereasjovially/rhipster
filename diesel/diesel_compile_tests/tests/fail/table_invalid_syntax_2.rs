#[macro_use] extern crate diesel;

table! {
     some wrong syntax
}
// error-pattern: Invalid `table!` syntax. Please see the `table!` macro docs for more info.

fn main() {}
