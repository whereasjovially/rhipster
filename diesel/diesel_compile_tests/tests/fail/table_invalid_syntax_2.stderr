error: Invalid `table!` syntax. Please see the `table!` macro docs for more info.
 --> $DIR/table_invalid_syntax_2.rs:3:1
  |
3 | / table! {
4 | |      some wrong syntax
5 | | }
  | |_^
  |
  = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)
